framework/src/base/MaxQpsThread.C

```diff
@@ -63,26 +65,26 @@ MaxQpsThread::operator() (const ConstElemRange & range)
       // We cannot mess with the FE objects in Assembly, because we might need to request second derivatives
       // later on. If we used them, we'd call reinit on them, thus making the call to request second
       // derivatives harmful (i.e. leading to segfaults/asserts). Thus, we have to use a locally allocated object here.
-      FEBase * fe = FEBase::build(dim, fe_type).release();
+      UniquePtr<FEBase> fe (FEBase::build(dim, fe_type));
 
       // figure out the number of qps for volume
-      QBase * qrule = QBase::build(_qtype, dim, _order).release();
-      fe->attach_quadrature_rule(qrule);
+      UniquePtr<QBase> qrule (QBase::build(_qtype, dim, _order));
+      fe->attach_quadrature_rule(qrule.get());
```

Comment:
Probably not, but I think it is cleaner and this isn't a time sensitive code or anything. We've been slowing getting rid of our raw pointers in some parts of MOOSE. Since I was in here anyway, I figured, why not?